### ISSUE TITLE
Add Parachain status check to pre-flight.

### ISSUE THREAD

**Issue by gianfra-t** [pendulum-chain/vortex#319]
### Description and potential addition

Recently an off-ramp failed at the `RequestRedeem` stage due to the Oracle being offline, and the redeem extrinsic failing with `security.ParachainNotRunning`. Furthermore, since the extrinsic was executed but failed, there is no way to recover these funds by re-executing it again.

This can actually be checked on initialization of the app (or in the backend and append to the status), and avoid starting off-ramps that will fail for certain.

**Comment by prayagd** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pendulum-6408a0365cda59c57d39bee5/issues/pendulum-chain/vortex/319?planning-poker) with Zenhub @ebma @gianfra-t @Sharqiewicz

**Comment by prayagd** [comment]
@vadaynujra should we priortize this, i consider this important as failing check would fail the offramp?

**Comment by ebma** [comment]
Please do, I also think it's important.

**Comment by vadaynujra** [comment]
@prayagd makes sense to prioritise this one given the low effort. Could you also confirm (preferably with a screenshot), what the user would see if the checks fail, today?

**Comment by ebma** [comment]
@vadaynujra we would just add this to the set of current checks and thus, it would also show the 'Application initialization failed...' error message, without giving any specific hint.

**Comment by vadaynujra** [comment]
Understood, thanks @ebma

**Comment by prayagd** [comment]
This ticket is depriortized for the current sprint

**Comment by prayagd** [comment]
@ebma as per my understanding this is already done right? we are showing the digital traffic jam error already

**Comment by prayagd** [comment]
Got it
